android/templates/build/ti.constants.gradle

```diff
@@ -7,9 +7,9 @@
 
 project.ext {
 	tiAndroidXAppCompatLibVersion = '1.5.0'
-	tiAndroidXCoreLibVersion = '1.8.0'
-	tiAndroidXFragmentLibVersion = '1.5.2'
-	tiMaterialLibVersion = '1.6.1'
+	tiAndroidXCoreLibVersion = '1.9.0'
+	tiAndroidXFragmentLibVersion = '1.5.5'
+	tiMaterialLibVersion = '1.6.0'
```

Comment:
Downgrade?

Reply:
sorry, my bad. Change the `7` to `6` and forgot to increase the last digit again :smile: